src/components/Card/ScheduleCard.js

```diff
@@ -20,13 +20,13 @@ const ScheduleCard = ({
       style={{
         minHeight: '510px',
         minWidth: '336px',
-        maxHeight: '100%',
+        height: '100%',
         maxWidth: '576px',
       }}
       round="medium"
       overflow="hidden"
     >
-      <Box direction="column">
+      <Box direction="column" className="HELLO ONE">
```

Comment:
should we take these out ? Im guessing you were just inspecting and searching for classNames

Reply:
woops

Reply:
done